src/app/pages/classes-page/classes-page.component.scss

```diff
@@ -0,0 +1,222 @@
+@use "../../../import" as *;
+
+.circle{
+    height: 30px;
+    width: 30px;
+    border-radius: 30px;
+    background-color:#DDB375;
+    color: black;
+    display:flex;
+    align-items: center;
+    justify-content:center;
+}
+
+.text-description{
+    margin-left:5%;
+    margin-right:5%;
+    margin-top:2%;
+}
+
+.circleClasses{
+    height: 30px;
+    width: 30px;
+    border-radius: 30px;
+    background-color:#DDB375;
+    color: black;
+    display:flex;
+    align-items: center;
+    justify-content:center;
+    margin-top:8%;
+    justify-content: center;
+    align-items: center;
+}
+
+.background-image
+{
+  background-image: url(../../../assets/background_pictures/backgorund_stat.jpg);
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-position: center center;
+  background-attachment: fixed;
+}
+
+.content-page
+{
+    display:flex;
+    flex-direction: row;
+    justify-content: space-around;
+    padding-bottom:7%;
+}
+
+.selectClass{
+    background-color: #202229;
+    filter: opacity(90%);
+    margin: $globalPaddingTop 0 2rem 0;
+    border-radius: 10px;
```

Comment:
use the appropriate variable in _import.scss $textColor: black;
Replace color with variable => color :  $textColor